MobileClient/Client/Pages/Homepage.razor.cs

```diff
@@ -14,11 +14,23 @@ protected override async Task OnInitializedAsync()
     {
         await base.OnInitializedAsync().ConfigureAwait(false);
         items = await GetItemsAsync().ConfigureAwait(false);
-    }
+
+        foreach (var item in items)
+        {
+            item.Pictures = await GetItemPicturesAsync(item.Id).ConfigureAwait(false);
```

Comment:
Wir sollten vlt eine Methode Server seitig bereitstellen, der das CoverPic direkt zurück gibt. Das wäre in dem Falle Performanter.